state/export_test.go

```diff
@@ -64,6 +64,10 @@ func (p *Pruner) PruneBlockIndexerToRetainHeight(lastRetainHeight int64) int64 {
 	return p.pruneBlockIndexerToRetainHeight(lastRetainHeight)
 }
 
+func (p *Pruner) PruneBlocksToHeight(height int64) (uint64, int64, error) {
```

Comment:
Just rename the private method?

Reply:
@thanethomson , @mzabaluev , we used to favor this approach as to avoid exposing functions that we use internally if they are exposed just for testing. I don't mind altering the code but want to make sure we are all on the same page.

Reply:
Ah I see. Yes @mzabaluev if it's just exposed for testing purposes, we basically create a public wrapper method in a `_test.go` file. These `_test.go` files are not built when building the binary, but are built when running tests.